### ISSUE TITLE
CanvasRenderingContext2D: drawImage() method: takes natural height and width into account docs incomplete.

### ISSUE THREAD

**Issue by sarojregmi200** [mdn/content#35141]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawImage

### What specific section or headline is this issue about?

Parameters section.

### What information was incorrect, unhelpful, or incomplete?

The information is incomplete.
In the Parameters section of the docs inside the image parameter.

> "An element to draw into the context. The specification permits any canvas image source, specifically, an [HTMLImageElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement), an [SVGImageElement](https://developer.mozilla.org/en-US/docs/Web/API/SVGImageElement), an [HTMLVideoElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLVideoElement), an [HTMLCanvasElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement), an [ImageBitmap](https://developer.mozilla.org/en-US/docs/Web/API/ImageBitmap), an [OffscreenCanvas](https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas), or a [VideoFrame](https://developer.mozilla.org/en-US/docs/Web/API/VideoFrame)."

### What did you expect to see?

A note saying that the when [HTMLImageElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement), an [SVGImageElement](https://developer.mozilla.org/en-US/docs/Web/API/SVGImageElement) are used the [naturalWidth](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/naturalWidth) and [naturalHeight](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/naturalHeight) is taken into account not the height and width set by the browser.

### Do you have any supporting links, references, or citations?

No, Need to do some research.

### Do you have anything more you want to share?

> I was stuck on a problem for 1/2 hrs while building a image cropper component.
> I don't know the actual reason why that is would love to research on it and submit a pr consisting the fix.

**Steps to reproduce:**
- Add a image with some higher resolution. 
- Scale it down.
- Take the reference of that image and then render the image in the canvas.

**What will you observe?** 
You will see that the image appears to be of bigger size that the scaled down one. 
Which tripped me off, anyways I don't know why that behavior is and would love to research on the topic.

**Note:** If this issue breaks any rules or is not correct I am open for criticism and would love to know where I fell short.

**Comment by Josh-Cena** [comment]
@sarojregmi200 Could you give the code used to reproduce said issue?

**Comment by sarojregmi200** [comment]
I will respond with a Example in a sandbox as soon as possible.

**Comment by sarojregmi200** [comment]
Hey, @Josh-Cena sorry about the delay I completely forgot about it.

I have created a [repo](https://github.com/sarojregmi200/context-api-reproducable-example.git) with the sample code and here is the [live](https://sarojregmi200.github.io/context-api-reproducable-example/) example of the above mentioned issue. 

> So, let me rephrase the above issue in a more understandable manner. In the documentation for the context api's [drawImage](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawImage) method while using `HTMLImageElement` or any other `DOM` media element as it's source. It is not clearly specified that even if the source is a `DOMElement` reference the dimensions of the actual media source is taken into account. 


> Due to this missing piece of information I have found myself in 1-2 hrs of debugging state.
